tox.ini

```diff
@@ -21,3 +21,14 @@ deps =
 
 ; Pass $PWD as it is when tox is invoked to pytest (used to find hlwm binaries)
 passenv = PWD
+
+[testenv:flake8]
+deps =
+    flake8
+commands = flake8 .
+
+[flake8]
+ignore =
+    E501, # line too long
+    W503, # incompatible with pep8.. "Line break occurred before a binary operator"
```

Comment:
A bit of context: For years, pep8 required line breaks to be after binary operators, then [it changed](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) to instead require the opposite. Thus, flake8 (or rather pycodestyle, which flake8 uses) has both checks, but [disables](http://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-ignore) W503 by default.

Setting a custom `ignore = E501` overrides the default ignores, which is why this needs to be re-added.

Reply:
Thanks for the explanation!